src/ops/conv_2d.cc

```diff
@@ -1074,6 +1081,96 @@ tl::optional<RecordFormatter> Conv2D::as_dot() const {
   return rr;
 }
 
+// TODO: refactor it
```

Comment:
Is this still an issue? Or was the refactor to move it from the `.cu` file into this one?

Reply:
Honestly I don't remember so I'll get rid of it